understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
-                     <p class="note-title marker">Note</p>
-                     <p>For this Success Criterion, "the same order relative to other page content" can be
-                        thought of as how the content is ordered when the page is serialized. The visual position
-                        of a help mechanism is likely to be consistent across pages for the same page variation
-                        (e.g., CSS break-point). The user can initiate a change, such as changing the page's
-                        zoom or orientation, which may trigger a different page variation. This criterion
-                        is concerned with relative order across pages displayed in the same page variation
-                        (e.g., same zoom level and orientation).
+                     <p class="note-title marker">注記</p>
+                     <p>この達成基準における「他のページコンテンツに対して相対的に同じ順序」は、ページが線形化されたときのコンテンツの並べ方と考えることができる。ヘルプのメカニズムの視覚的な位置は、ページの同じバリエーション (例えば CSS のブレークポイント) に対してページ間で一貫している可能性が高い。利用者は、ページのズーム、ページの向きを変えるなどの変更を発生させることができ、これはページのバリエーションが変化するきっかけになる可能性がある。この達成基準で扱うのは、同じバリエーション (例えば、同じズームレベル、向き) で表示されるページ同士の間における、相対的な順序である。
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>簡潔に</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make it easier to find help and support.</dd>
+                  <dt>目標</dt>
+                  <dd>ヘルプ及びサポートを見つけやすくする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Put help in the same place when it is on multiple pages.</dd>
+                  <dt>何をするか</dt>
+                  <dd>ヘルプが複数のページにある場合は、同じ場所にヘルプを配置する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who need help can find it more easily if it's in the same place.</dd>
+                  <dt>なぜ重要か</dt>
+                  <dd>同じ場所にあれば、ヘルプを必要とする人は、より簡単に見つけることができる。</dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure users can find help for completing
-                  tasks on a Web site, when it is available. When the placement of the help mechanism
-                  is kept consistent across a set of pages, users looking for help will find it easier
-                  to identify. This is distinct from interface-level help, such as contextual help,
-                  features like spell checkers, and instructional text in a form. 
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ウェブサイト上で作業を完了するためのヘルプが利用可能な場合に、そのヘルプを利用者が確実に見つけられるようにすることである。ヘルプメカニズムの配置が一連のページ全体で一貫していると、ヘルプを探している利用者が見つけやすくなる。これは、コンテキストヘルプ、スペルチェッカーのような機能、フォーム内の説明テキストなどのインターフェースレベルのヘルプとは異なる。
```

Comment:
「一連のページ全体」原文では a set of pages で、おそらく「a set of web pages (Webページ一式)」と同義かと思います。

```suggestion
               <p>この達成基準の意図は、ウェブサイト上で作業を完了するためのヘルプが利用可能な場合に、そのヘルプを利用者が確実に見つけられるようにすることである。ヘルプメカニズムの配置がウェブページ一式を通じて一貫していると、ヘルプを探している利用者が見つけやすくなる。これは、コンテキストヘルプ、スペルチェッカーのような機能、フォーム内の説明テキストなどのインターフェースレベルのヘルプとは異なる。
```

Reply:
@caztcha 

ありがとうございます。

> 「一連のページ」原文では a set of pages で、おそらく「a set of web pages (ウェブページ一式)」と同義かと思います。

ここについては、達成基準訳の

> ウェブページ一式の中にある複数のウェブページで繰り返されている

原文の
> on multiple Web pages within a set of Web pages

ことをもしかしたら指しているのかもしれない…と思っているところではあるのですがいかがでしょう…？

Reply:
@momdo はい、私もそう思います！

Reply:
ありがとうございます。であれば、（今の訳がよいかはさておき）「ウェブページ一式」と訳してしまうのはよろしくないと思うのです…。

Reply:
ああ、なるほど、理解しました。

ただ、「一連のページ全体で一貫している」だと、ページの中で一貫しているように読めてしまう恐れがあるかなと思いますので、「一連のページ群で一貫している」というのは、いかがでしょうか。

```suggestion
               <p>この達成基準の意図は、ウェブサイト上で作業を完了するためのヘルプが利用可能な場合に、そのヘルプを利用者が確実に見つけられるようにすることである。ヘルプメカニズムの配置が一連のページ群で一貫していると、ヘルプを探している利用者が見つけやすくなる。これは、コンテキストヘルプ、スペルチェッカーのような機能、フォーム内の説明テキストなどのインタフェースレベルのヘルプとは異なる。
```

Reply:
@caztcha 

ありがとうございます。いただいた案を取り込みます。